examples/quantified/strncmp/strncmp.c

```diff
@@ -0,0 +1,22 @@
+#include "klee/klee.h"
+#include <stdio.h>
+
+int main() {
+        char a[5], b[5];
+        klee_make_symbolic(a, sizeof(a), "a");
+        klee_make_symbolic(b, sizeof(b), "b");
+        int n = 5, res = strncmp(a, b, n);
+
+        // klee_assume(res == 0);
```

Comment:
A general comment about tests. 
For each test, give clear comments saying why the test should fail. 
Also, use multi-line comments, so the entire test can be commented/uncommented at once